packages/api/lib/reconciliationReport.js

```diff
@@ -1,14 +1,27 @@
 'use strict';
 
 const { removeNilProperties } = require('@cumulus/common/util');
+const { constructCollectionId } = require('@cumulus/message/Collections');
 const { deconstructCollectionId } = require('./utils');
+
+/**
+ * Prepare a list of collectionIds into an _id__in object
+ *
+ * @param {Array<string>} collectionIds - Array of collectionIds in the form 'name___ver'
+ * @returns {Object} - object that will return the correct terms search when
+ *                     passed to the query command.
+ */
+function searchParamsForCollectionIdArray(collectionIds) {
+  return { _id__in: collectionIds.join(',') };
```

Comment:
This feels like it's in the wrong place.  I don't think I should be writing these query objects here, but I don't know where they should do otherwise.

Reply:
I guess it's all getting removed eventually/soon anyway?